prime-router/src/main/kotlin/history/azure/ReportFileFunction.kt

```diff
@@ -161,10 +162,15 @@ abstract class ReportFileFunction(
         } catch (e: DataAccessException) {
             logger.error("Unable to fetch history for ID $id", e)
             return HttpUtilities.internalErrorResponse(request)
-        } catch (ex: IllegalStateException) {
-            logger.error(ex)
-            // Errors above are actionId or UUID not found errors.
-            return HttpUtilities.notFoundResponse(request, ex.message)
+        } catch (ex: IllegalStateException) { // actionId or UUID not found
```

Comment:
FYI @arnejduranovic looks like an extra commit snuck in here

Reply:
Whoops, I must've branched off Gabriel's branch instead of master. Reverted those commits on this branch.